recipe/meta.yaml

```diff
@@ -0,0 +1,195 @@
+  # torchaudio/io/_stream_writer.py::self._s.open(option)   throws a RuntimeError
+  # RuntimeError: Failed to write header:
+  # /tmp/tmpx2on2_qz/torchaudio_unittest.backend.dispatcher.smoke_test.SmokeTest.test_wav/test.wav (Function not implemented)
+  #
+  # since the save function constitutes one of the basic functionalities of torchaudio, it is best to skip the platform altogether
+  # as long as other backends (sox, libsndfile) are not available for s390x
+  skip: True  # [s390x]
+  script:
+    - export USE_FFMPEG=ON                         # [not win]
+    - set USE_FFMPEG=ON                            # [win]
+    - export BUILD_SOX=OFF                         # [not win]
+    - set BUILD_SOX=OFF                            # [win]
+    # On linux, Torchaudio's build system will find gnu openmp preferentially to other openmp implementations,
+    # and we can't keep gnu openmp out of the host environment due to issues with our openmp packages.
+    # That could lead to conflicts with other packages. Turn it off for linux.
+    - export USE_OPENMP=OFF                        # [linux]
+    - export USE_OPENMP=ON                         # [osx]
+    - set USE_OPENMP=ON                            # [win]
+    - export BUILD_TORCHAUDIO_PYTHON_EXTENSION=ON  # [not win]
+    - set BUILD_TORCHAUDIO_PYTHON_EXTENSION=ON     # [win]
+    - export FFMPEG_ROOT=${PREFIX}                 # [not win]
+    - set FFMPEG_ROOT=%LIBRARY_PREFIX%             # [win]
+    - {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  missing_dso_whitelist:
+    - "**/libc10.so"                # [linux]
+    - "**/libtorch_cpu.so"          # [linux]
+    - "**/libtorch_python.so"       # [linux]
+    - "**/libtorchaudio_ffmpeg.so"  # [linux or osx]
+    - "**/libtorchaudio.so"         # [linux or osx]
+    - "**/libc10.dylib"             # [osx]
+    - "**/libtorch_cpu.dylib"       # [osx]
+    - "**/libtorch_python.dylib"    # [osx]
+    - "**/torch_cpu.dll"            # [win]
+    - "**/c10.dll"                  # [win]
+    - "**/torch_python.dll"         # [win]
+    - "**/libtorchaudio_ffmpeg.pyd" # [win]
+    - "**/libtorchaudio.pyd"        # [win]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}
+    - cmake
+    - make      # [not win]
+    - patchelf  # [linux]
+  host:
+    - python
+    - setuptools
+    - wheel
+    - pip
+    - pytorch {{ compatible_pytorch }}
+    - pybind11 2.12.0
+    - ffmpeg 6.1.1
+    - llvm-openmp  # [osx]
+  run:
+    - python
+    # according to optional dependencies
+    # https://github.com/pytorch/audio/blob/v2.1.2/docs/source/installation.rst#optional-dependencies
+    # in this special case we want users of channel sfe140ed to be able to use the load and save functions, therefore
+    # at least ffmpeg (the mightiest of the backends) is specified as a run dependency.
```

Comment:
```suggestion
```
I would remove this block of comment. It's misleading since ffmpeg is optiona at build time. But if compiled with ffmpeg, it's mandatory.

Reply:
done.